api/v1beta1/types.go

```diff
@@ -1213,22 +1216,33 @@ func (network *OscNetwork) SetSubnetDefaultValue() {
 		var subnetKcpName string = DefaultSubnetKcpName
 		var subnetKwName string = DefaultSubnetKwName
 		var subnetPublicName string = DefaultSubnetPublicName
+		var subnetSubregionName string = DefaultSubregionName
+		if network.SubregionName != "" {
+			subnetKcpName = network.SubregionName
```

Comment:
what is the link between subnetName and subRegionName ?

Reply:
Yeah this is a mistake, i wil fix this bit of code